src/libraries/System.Net.Requests/src/System/Net/ServicePoint/ServicePointManager.cs

```diff
@@ -46,6 +46,8 @@ private static void ValidateSecurityProtocol(SecurityProtocolType value)
             }
         }
 
+        internal static TcpKeepAlive? KeepAlive { get; private set; }
```

Comment:
statics and nullables generally don't mix well, as there can be race conditions in non-atomically reading the bool for whether there's a value and the actual value.  I suggest making this TcpKeepAlive a reference type instead.